src/api/user.ts

```diff
@@ -0,0 +1,31 @@
+import { apiPost, apiGet } from './base';
+import { User, ApiError, FormData } from '../types';
+import { ERROR_MESSAGE } from '../utils/constants';
+
+export const authUser = async (
+  formData: FormData,
+  onError?: (error: ApiError) => void,
+) => {
+  try {
+    return await apiPost('user/login', formData, undefined, onError);
+  } catch (error: unknown) {
+    const Error = error as any;
```

Comment:
Not the first time I have noticed you use capital letters. Dont, please. It has nothing to do in there

Reply:
😮